docs/pyboard/quickref.rst

```diff
@@ -219,6 +219,23 @@ eg ``I2C(1)``.  Software I2C is also available by explicitly specifying the
 
 Note: for legacy I2C support see :ref:`pyb.I2C <pyb.I2C>`.
 
+I2S bus
+-------
+
+See :ref:`machine.I2S <machine.I2S>`. ::
+
+    from machine import I2S, Pin
+    
+    i2s = I2S(2, sck=Pin('Y6'), ws=Pin('Y5'), sd=Pin('Y8'), mode=I2S.TX, bits=16, format=I2S.STEREO, rate=44100, ibuf=40000) # create I2S object
+    i2s.write(buf)             # write buffer of audio samples to I2S device
+    
+    i2s = I2S(1, sck=Pin('X5'), ws=Pin('X6'), sd=Pin('Y4'), mode=I2S.RX, bits=16, format=I2S.MONO, rate=22050, ibuf=40000) # create I2S object
+    i2s.readinto(buf)          # fill buffer with audio samples from I2S device
+    
+PYBv1.0/v1.1 has one I2S bus with id=2.  
+PYBD-SFxW has two I2S busses with id=1 and id=2.  
```

Comment:
"buses"

Reply:
fixed in https://github.com/miketeachman/micropython/commit/0f7b59c92e8173d86f596097f9f8fc5ef3e3cfd5